### PR TITLE
redis: implement echo command

### PR DESCRIPTION
**Pull request by amoskong**
This patch implemented echo command, which return the string back to client.

Reference:
- https://redis.io/commands/echo

/CC @fastio @nyh

### CODE REVIEW

File: redis/commands/echo.hh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    bytes _str;
+public:
+    static shared_ptr<abstract_command> prepare(service::storage_proxy& proxy, request&& req);
+    echo(bytes&& name, bytes&& str)
+        : abstract_command(std::move(name))
+        , _str(std::move(str))
+    {
+    }
+
+    ~echo()
```
Comment (denesb) [inline redis/commands/echo.hh]: No need to declare this.

File: redis/commands/echo.hh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    {
+    }
+
+    future<redis_message> execute(service::storage_proxy&, redis_options&, servcie_permit) override;
```
Comment (denesb) [inline redis/commands/echo.hh]: We prepend `virtual` to overriden virtual method declarations.

File: redis/commands/echo.cc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+#include "types.hh"
+#include "service/storage_proxy.hh"
+#include "redis/options.hh"
+#include "service_permit.hh"
+
+namespace redis{
+namespace commands {
+
+shared_ptr<abstract_command> echo::prepare(service::storage_proxy& proxy, request&& req)
+{
```
Comment (denesb) [inline redis/commands/echo.cc]: Opening parens on the same line as the function signature.

File: redis/commands/echo.cc [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    if (req.arguments_size() != 1) {
+        throw wrong_arguments_exception(1, req.arguments_size(), req._command);
+    }
+    request seastar::make_shared<echo> (std::moved(req._command), std::move(req._args[0]);
+}
+
+future<redis_message> echo::execute(service::storage_proxy&, redis::redis_options&, service_permit)
+{
```
Comment (denesb) [inline redis/commands/echo.cc]: Here too.

File: redis/commands/echo.cc [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+#include "redis/commands/echo.hh"
+#include "seastar/core/shared_ptr.hh"
+#include "redis/request.hh"
+#include "redis/reply.hh"
+#include "types.hh"
+#include "service/storage_proxy.hh"
+#include "redis/options.hh"
+#include "service_permit.hh"
+
+namespace redis{
```
Comment (denesb) [inline redis/commands/echo.cc]: Space between `redis` and `{`.

**Comment by amoskong** [comment]
@denesb Thanks for your review, I just updated the patches.

**Pull request merged**